core/designsystem/src/main/kotlin/com/dhc/designsystem/calendar/DhcCalendarController.kt

```diff
@@ -15,8 +16,11 @@ class DhcCalendarController(
     var currentDate by mutableStateOf(initialData.initialDate)
         private set
 
+    var calendarMonthState by mutableStateOf(mapOf<LocalDate, DhcCalendarMonthData>())
+        private set
```

Comment:
![image](https://github.com/user-attachments/assets/3c3bf63e-38f6-4850-8fc8-46df932762bc)

```suggestion
    val calendarMonthState = mutableStateMapOf<LocalDate, DhcCalendarMonthData>()
```

P1 : mutableStateMapOf 의 맛을 한번 보시겠습니까? (지금 상태론 하나 바뀌면 저거 참조하는 모든 곳 바뀔 거 같음)